src/main/java/picard/sam/SamErrorMetric/ReadBaseStratification.java

```diff
@@ -37,6 +37,7 @@
 import picard.sam.util.Pair;
 import picard.sam.util.PhysicalLocation;
```

Comment:
are these all used?

Reply:
There's one that isn't used, I'll delete it